app-ios/Sources/AboutFeature/AboutView.swift

```diff
@@ -18,8 +18,21 @@ public struct AboutEnvironment {
     public init() {}
 }
 
-public let aboutReducer = Reducer<AboutState, AboutAction, AboutEnvironment> { _, _, _ in
-    return .none
+public let aboutReducer = Reducer<AboutState, AboutAction, AboutEnvironment> { _, action, _ in
+    @Environment(\.openURL) var openURL
```

Comment:
Please define this in environment ( `AboutEnvironment` ) 🙏🏼 
Best is to make it injectable from the outside. But it is okey for now 👍🏼